postgres_setup.sh

```diff
@@ -0,0 +1,2 @@
+psql -h localhost -U postgres -w -c "create database instascraper;"
+migrate -database 'postgres://postgres:password@localhost:5432/instascraper?sslmode=disable' -path db/migrations up
```

Comment:
You don't need to add this file, in #13 I add a similar setup thing that does a bit more